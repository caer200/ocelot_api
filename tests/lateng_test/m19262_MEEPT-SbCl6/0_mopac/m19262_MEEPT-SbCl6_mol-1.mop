CHARGES

 
Sb	3.445963 +1 5.960149 +1 14.044333 +1
Cl	3.328687 +1 8.300785 +1 13.799303 +1
Cl	3.582758 +1 3.594710 +1 14.309412 +1
Cl	5.817873 +1 6.120574 +1 14.035062 +1
Cl	1.094837 +1 5.825313 +1 14.033870 +1
Cl	3.486662 +1 5.685359 +1 11.699356 +1
Cl	3.437981 +1 6.159549 +1 16.395229 +1
