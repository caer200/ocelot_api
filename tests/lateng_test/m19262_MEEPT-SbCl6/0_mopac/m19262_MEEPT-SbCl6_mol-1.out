 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 22:31:20 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Sb          3.44596300  *   5.96014900  *  14.04433300  *
     2      Cl          3.32868700  *   8.30078500  *  13.79930300  *
     3      Cl          3.58275800  *   3.59471000  *  14.30941200  *
     4      Cl          5.81787300  *   6.12057400  *  14.03506200  *
     5      Cl          1.09483700  *   5.82531300  *  14.03387000  *
     6      Cl          3.48666200  *   5.68535900  *  11.69935600  *
     7      Cl          3.43798100  *   6.15954900  *  16.39522900  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Sb          3.4460      5.9601     14.0443
     2        Cl          3.3287      8.3008     13.7993
     3        Cl          3.5828      3.5947     14.3094
     4        Cl          5.8179      6.1206     14.0351
     5        Cl          1.0948      5.8253     14.0339
     6        Cl          3.4867      5.6854     11.6994
     7        Cl          3.4380      6.1595     16.3952

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: Sb Cl6  =     7 atoms



      MOLECULAR POINT GROUP   :   Ci  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1         Sb      2      3      4      5      6      7
      2         Cl      1
      3         Cl      1
      4         Cl      1
      5         Cl      1
      6         Cl      1
      7         Cl      1

     Ion Atom No.  Type    Charge
 
    1       1     Sb       -1

          COMPUTED CHARGE ON SYSTEM: -1

          COMPUTED CHARGE ON SYSTEM:  -1

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
